mobile/library/cc/engine_builder.h

```diff
@@ -89,6 +89,11 @@ class EngineBuilder {
   // For example if the runtime guard is `envoy.reloadable_features.use_foo`, the guard name is
   // `use_foo`.
   EngineBuilder& addRuntimeGuard(std::string guard, bool value);
```

Comment:
I wish we could rename this to `addRelodableRuntimeGuard` :)

Reply:
yeah, in a follow up :)  i wanted to do the same but it'll break existing code.  i'll probably add addReloadableRuntimeGuard, then when the code is switched over to using it, delete addRuntimeGuard